readme.txt

```diff
@@ -238,6 +238,7 @@ Remember to always make a backup of your database and files before updating!
 * Fix - Correct docblocks in the file `Tribe\Events\Views\V2\Views\Traits\Breakpoint_Behavior.php` - credit goes to @IanDelMar
 * Fix - Add missing closing tags on some admin pages to ensure valid HTML markup. [TEC-4807]
 * Fix - Fix an issue where the import screen broke when the import limit type was set to "date range". [TECTRIA-103]
+* Fix - Changed `else if` to `if` in `filter_posts_join()` function to avoid database error. Credit goes to @datadiver0x0. [ECP-1562]
```

Comment:
Changelog needs to be plain English. Not too technical.
```suggestion
* Fix - Changed logic within the custom tables query to avoid database error. Credit goes to @datadiver0x0. [ECP-1562]
```